cmd/codegen/gengo.go

```diff
@@ -278,15 +299,17 @@ import "unsafe"
 				args = append(args, fmt.Sprintf("%s %s", a.Name, argType))
 
 				shouldGenerate = true
+				continue
 			}
 
-			if isEnum(a.Type) {
-				args = append(args, fmt.Sprintf("%s %s", a.Name, a.Type))
+			if isEnum(trimImGuiPrefix(a.Type)) {
+				args = append(args, fmt.Sprintf("%s %s", a.Name, trimImGuiPrefix(a.Type)))
 				argWrappers = append(argWrappers, argOutput{
```

Comment:
like above: define a variable in if-statement initialization to drop duplicated `trimImGuiPrefix` calls